### DISCUSSION TITLE
Good afternoon, help me install and configure the LIAP package correctly

### DISCUSSION THREAD

**Discussion by astmania** [imdhemy/laravel-in-app-purchases#330] [Q&A]
I have an app on ios and Android, but I don't understand how to use LIAP in laravel to connect! Someone help me if I need to pay, I will pay if you help

**Comment by cosmastech** [comment]
Do you already have a Laravel project tied to these apps?

What is it you want with the data from App Store and play store?

**Reply by astmania** [reply]
Yes, there is already an existing project on Laravel, I need to check purchases from the current application, the application itself on two iOS and android systems

**Reply by astmania** [reply]
I need a person who can implement all the events on two systems, it will be great if you then explain how you made the implementation in order to further serve it
